datadog_checks_base/datadog_checks/base/utils/db/statement_metrics.py

```diff
@@ -104,6 +105,33 @@ def compute_derivative_rows(self, rows, metrics, key):
         return result
 
 
+def merge_duplicate_rows(rows, metrics, key):
```

Comment:
Can make this private (`_merge_duplicate_rows`) since the intention is not to expose it.

Reply:
This makes sense! Here's a follow-up question: when making __merge_duplicate_rows_ private, should I remove the direct test I had added? I think that's the common pattern to test the public methods and skip the private ones (and we have coverage via `test_compute_derivative_rows_with_duplicates`) but let me know if you think I should keep `test_merge_duplicate_rows`.

Reply:
Made it private and removed the test I had added for the now private method.